src/Microsoft.AspNet.Mvc.ModelBinding/Binders/BindingSourceModelBinder.cs

```diff
@@ -59,6 +59,10 @@ protected BindingSourceModelBinder([NotNull] BindingSource bindingSource)
         /// <returns>
         /// A <see cref="Task"/> which will complete when model binding has completed.
         /// </returns>
+        /// <remarks>
+        /// Other model binders will never run if this method is called. Return <c>null</c> to skip other model binders
+        /// but allow higher-level handling e.g. falling back to empty prefix.
+        /// </remarks>
```

Comment:
no, I'm saying this specific `abstract` method leaves the `override` no way to let other binders run.  in your example, the `BodyModelBinder` implementation won't call this method.  it handles exclusivity before calling `BindModelCoreAsync`.